app/controllers/brexit_landing_page_controller.rb

```diff
@@ -1,16 +1,28 @@
 class BrexitLandingPageController < ApplicationController
+  include BrexitLandingPageTestable
```

Comment:
I'd suggest we actually move the code from this include into the controller. I'm not sure if we'd intend to use it in any other controllers and I think we could reduce the amount of code in doing so.

We could probably have:
```
helper_method :page_variant
after_action { page_variant.configure_response(response) }

# ...

def page_variant
  @page_variant ||= begin
    test = GovukAbTesting::AbTest.new(
             "BrexitLandingPageTest",
             dimension: CUSTOM_DIMENSION,
             allowed_variants: %w(A B),
             control_variant: "A",
           )
    test.requested_variant(request.headers)
  end
end
```

which might make it feel a bit simpler.

Reply:
Sure, I've made a few small changes to your suggestion but have relocated and cut down the code... success!